File: Python/programmers/etc/level1/옹알이2.py
def solution(babbling):
    answer = 0
    for b in babbling:
        for r in ["aya", "ye", "woo", "ma"]:
            if r*2 not in b:
                b = b.replace(r, " ")
        if len(b.strip()) == 0:
            answer += 1
    return answer


babbling = ["aya", "yee", "u", "maa"]
print(solution(babbling)) # 1

babbling = ["ayaye", "uuu", "yeye", "yemawoo", "ayaayaa"]
print(solution(babbling)) # 2

File: Java/programmers/etc/level1/옹알이2.java
package CodeTest.Java.programmers.etc.level1;

public class 옹알이2 {
    public int solution(String[] babbling) {
        int answer = 0;
        
        for (String b : babbling) {
            b = b.replaceAll("ayaaya|yeye|woowoo|mama","1");
            b = b.replaceAll("aya|ye|woo|ma"," ");
            b = b.replaceAll(" ","");
            if(b.equals("")) answer++;
        }
        
        return answer;
    }
}
